apps/wallet-mobile/src/Dashboard/DashboardNavigator.tsx

```diff
@@ -17,34 +20,49 @@ export const DashboardNavigator = () => {
   const walletName = useWalletName(wallet)
   const strings = useStrings()
 
+  const {networkId} = useSelectedWallet()
+  const manager = useMemo(
+    () =>
+      governanceManagerMaker({
+        walletId: wallet.id,
+        networkId,
+        api: governanceApiMaker({networkId}),
+        cardano: CardanoMobile,
+        storage: AsyncStorage,
```

Comment:
shouldn't it use `.join` to return the desired storage folder or consume from `useStorage` the current `<root>` of the wallet? otherwise `.setItem` will store by default int `/` of the storage. 🤔